cmds/mkfifo/mkfifo.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2013-2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"flag"
+	"log"
+	"syscall"
+
+	"github.com/u-root/u-root/pkg/mkfifo"
+)
+
+var (
+	defaultMode = syscall.S_IRUSR | syscall.S_IWUSR | syscall.S_IRGRP | syscall.S_IWGRP | syscall.S_IROTH | syscall.S_IWOTH
```

Comment:
SHouldn't this be a const?

Reply:
Have you consider using 660 as the default mode? It would be tad bit more secure.